lib/components/narrative/metro/metro-itinerary.tsx

```diff
@@ -339,7 +347,7 @@ class MetroItinerary extends NarrativeItinerary {
             )}
             {!mini && (
               <ItineraryGrid className="itin-grid">
-                <Routes enableDot={enableDot}>
+                <Routes aria-hidden enableDot={enableDot}>
```

Comment:
I would at least add a comment here that we need to add an aria-label to the parent